list_resources.sh

```diff
@@ -1,19 +1,23 @@
-#!/bin/bash
-specs_dir="./specs"
+#!/bin/sh
 
-function findResourceDir(){
-    for file in `ls $1`
-    do
-    	work_dir=$1"/"$file
-        if [ -d $work_dir ]
-        then
-        	if [ -n "$(git status -s $work_dir)" ] && [ -f $work_dir"/api.yaml" ] \
-        		&& ([ -f $work_dir"/ansible.yaml" ] || [ -f $work_dir"/terraform.yaml" ]); then
-				echo ${work_dir:8}		# remove the directory prefix './specs/'
+findResourceDir(){
+	for file in `ls $1`
+	do
+		work_dir=$1"/"$file
+		if [ -d $work_dir ]
+		then
+			if [ -n "$(git status -s $work_dir)" ] && [ -f $work_dir"/api.yaml" ] \
+				&& ([ -f $work_dir"/ansible.yaml" ] || [ -f $work_dir"/terraform.yaml" ]);
+			then
+				# remove the directory prefix './specs'
+				expr substr "$work_dir" 9 "${#work_dir}"
 			fi
-            findResourceDir $work_dir	# find resource directory recursively
-        fi
-    done
+			# find resource directory recursively
+			findResourceDir $work_dir
```

Comment:
No need to do it recursively, we only try first level under the root folder of specs.

Reply:
Removed.